StripeFinancialConnections/StripeFinancialConnections/Source/Native/NativeFlowController.swift

```diff
@@ -193,7 +203,7 @@ extension NativeFlowController {
 
     private func didSelectAnotherBank() {
         if dataManager.manifest.disableLinkMoreAccounts {
-            closeAuthFlow(showConfirmationAlert: false, error: nil)
+            closeAuthFlow(error: nil)
```

Comment:
a key thing to notice (+ double check) in this refactor is that all call sites of `closeAuthFlow` have `showConfirmationAlert: false` EXCEPT one case...

this means that `CloseConfirmationAlertHandler` needs to only be used in that ONE case